PostGreKeyRotation/Deployment/pg-keyrotation-pipeline.yml

```diff
@@ -0,0 +1,179 @@
+trigger:
+- none
+
+stages:
+- stage: KeyRotation
+  jobs:
+  - job: KeyRotation_Job
+    pool:
+      vmImage: 'ubuntu-latest'
+    variables:
+    - group: key-rotation-variables
+    steps:
+    - task: Bash@3
+      displayName: 'Installing PostgreSQL client'
+      inputs:
+        failOnStderr: false
+        targetType: 'inline'
+        script: |
+          sudo apt-get install postgresql-client -y
+
+    - task: AzureCLI@2
+      displayName: 'Activate Inactive Role'
+      inputs:
+        failOnStandardError: true
+        azureSubscription: '<azure-subscription-id>'
+        scriptType: 'bash'
+        scriptLocation: 'scriptPath'
+        scriptPath: '$(System.DefaultWorkingDirectory)/PostGreKeyRotation/Deployment/scripts/pg-rotate-keys.sh'
+        workingDirectory: '$(System.DefaultWorkingDirectory)/PostGreKeyRotation/Deployment/scripts'
+        arguments: '"$(currentActiveRole)"'
+
+- stage: DeployStaging
+  dependsOn:
+  - KeyRotation
+  condition: succeeded('KeyRotation')
+  jobs:
+  - job: DeployStaging
+    pool:
+      vmImage: 'ubuntu-latest'
+    variables:
+    - group: key-rotation-variables
+    - group: pipeline-variables
+    steps:
+    - task: Kubernetes@1
+      displayName: 'Kubernetes Login'
+      inputs:
+        connectionType: 'Kubernetes Service Connection'
+        kubernetesServiceEndpoint: 'Kubernetes'
+        command: 'login'
+
+    - task: HelmInstaller@1
+      displayName: 'Install Helm'
+      inputs:
+        helmVersionToInstall: 'latest'
+
+    - task: AzureCLI@2
+      displayName: 'Deploy Staging'
+      inputs:
+        failOnStandardError: true
+        azureSubscription: '<azure-subscription-id>'
+        scriptType: 'bash'
+        scriptLocation: 'scriptPath'
+        scriptPath: '$(System.DefaultWorkingDirectory)/PostGreKeyRotation/Deployment/scripts/pg-deploy-staging.sh'
+        workingDirectory: '$(System.DefaultWorkingDirectory)/PostGreKeyRotation/Deployment/scripts'
+        arguments: '"$(currentActiveRole)" "$(buildTag)"'
```

Comment:
We had to add two more parameters here in the script: DBHOSTNAME and DBNAME

Reply:
Done